File: abc/abc156/a.cpp
// https://atcoder.jp/contests/abc156/submissions/26224075

#include <bits/stdc++.h>
using namespace std;
int main() {
  int n, r;
  cin >> n >> r;
  if(n < 10) cout << r + (100*(10-n));
  else cout << r;
}

File: abc/abc156/a.java
// https://atcoder.jp/contests/abc156/submissions/41451844

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    int r = stdIn.nextInt();
    
    if(n >= 10) {
      System.out.println(r);
    } else {
      System.out.println(r + 100 * (10 - n));
    }
  }
}
